### ISSUE TITLE
render function for grid column ran twice

### ISSUE THREAD

**Issue by james-yong** [vitmalina/w2ui#2086]
**Short description**
Custom render function for grid column is run twice instead of once, per row index and col_index.
Affects both master and 1.5 versions.

**What is current behavior**
> [Log] list::0::8 (test.html, line 65)
[Log] list::1::8 (test.html, line 65)
[Log] list::2::8 (test.html, line 65)
[Log] list::3::8 (test.html, line 65)
[Log] list::4::8 (test.html, line 65)
[Log] list::5::8 (test.html, line 65)
[Log] list::6::8 (test.html, line 65)
[Log] list::7::8 (test.html, line 65)
[Log] list::8::8 (test.html, line 65)
[Log] list::0::8 (test.html, line 65)
[Log] list::1::8 (test.html, line 65)
[Log] list::2::8 (test.html, line 65)
[Log] list::3::8 (test.html, line 65)
[Log] list::4::8 (test.html, line 65)
[Log] list::5::8 (test.html, line 65)
[Log] list::6::8 (test.html, line 65)
[Log] list::7::8 (test.html, line 65)
[Log] list::8::8 (test.html, line 65)

**What is desired behavior**
> [Log] list::0::8 (test.html, line 65)
[Log] list::1::8 (test.html, line 65)
[Log] list::2::8 (test.html, line 65)
[Log] list::3::8 (test.html, line 65)
[Log] list::4::8 (test.html, line 65)
[Log] list::5::8 (test.html, line 65)
[Log] list::6::8 (test.html, line 65)
[Log] list::7::8 (test.html, line 65)
[Log] list::8::8 (test.html, line 65)

**Link to jsfidle/codepan with sample code**

> { field: 'list', text: 'list', size: '50%', sortable: true, resizable: true,
                editable: { type: 'list', items: people, showAll: true },
                render: function (record, index, col_index) {
                    console.log(`list::${index}::${col_index}`);
                    var html = this.getCellValue(index, col_index);
                    return html || '';
                }
            },

**Steps to reproduce or sample** 
Copy the code from http://w2ui.com/web/demos/#/grid/21
Add console.log(`list::${index}::${col_index}`); to line 65.
Check the browser console after loading the page.

**Comment by mpf82** [comment]
ACK, calling w2ui.grid.refresh() calls the render function twice.

Fiddle: http://jsfiddle.net/Lqac1y52/

Needs further investigation.

**Comment by vitmalina** [comment]
I do not see it run twice int he fiddle provided above. Here is console.log I see
<img width="148" alt="Screen Shot 2021-10-04 at 8 31 29 AM" src="https://user-images.githubusercontent.com/561995/135798977-b11f02e1-913f-4962-8e93-3d66630cb599.png">
Which is correct. One time per each row. 

May be there is a second refresh call somewhere. For example after it returns from the server?

**Comment by mpf82** [comment]
True, I no longer see it rendering twice either, but it was definitely there.

However, the renderer seems to be executed twice when the grid is initially loaded, so there might be an unnecassry call nonetheless that might be worth investigating.
